### ISSUE TITLE
Sanitizer 

### ISSUE THREAD

**Issue by fervaz** [ctavan/express-validator#420]
Hi! 
I'm using express-validator with node v8.4.0 / npm v5.4.1 and the sanitizer is not working for me.

I have tried: `check('field').toFloat()` and `sanitize('field').toFloat` but I get an error: `TypeError: check(...).toFloat is not a function`. What I'm missing?
 
When I require `check` and `filter` and then log it's variables, then I see:

`const check = require('express-validator/check')`:
```
{ check: [Function],
  body: [Function],
  cookie: [Function],
  header: [Function],
  param: [Function],
  query: [Function],
  validationResult: [Function],
  oneOf: [Function] }
```

`const validator = require('express-validator/filter')`:
```
{ matchedData: [Function] }
```
It's correct? Where are the sanitizer?

Thank's.

**Comment by gustavohenke** [comment]
These are not released yet!
Sorry for making y'all wait 🕐 , I'll let you know when these changes get published to npm!

**Comment by gustavohenke** [comment]
This is now released with v4.2.0 🚢 

Full list of released features: https://github.com/ctavan/express-validator/releases/tag/v4.2.0
